_summaries/glorot2011deep_1.md

```diff
@@ -1,19 +1,77 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{glorot2011deep}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishank-arora
+score: 9
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+
+  * First paper (I believe) to prove using rectifier function ($$max(0,  x)$$) better than $$\textit{logistic sigmoid}$$ or $$\textit{hyperbolic tangent}$$ activation functions. Approaches this from the field of computational neuroscience. 
```

Comment:
on GitHub the formulas are not displayed correctly for me. I'm not sure if this is a formatting issue or if it just doesn't work in GitHub's viewer.